README.md

```diff
@@ -33,3 +33,18 @@ The Raspberry Pi hardware I2C does not support clock stretching. You have two op
 
 You may need to add your user to the i2c group, e.g. `sudo usermod -aG i2c nvidia`. It should just work after that.
 
+# ROS2
```

Comment:
Do we want to say something like:

output topics and service calls provided are the same as for the ROS driver

Reply:
Also, up on or before line 3 in this file we should say something like:

for the ROS2 node, follow the instructions in the ROS2 section below

Reply:
fixed.